tests/sklearn/test_sklearn_models.py

```diff
@@ -1153,35 +1146,36 @@ def test_correctness_with_sklearn(
     )
 
 
-@pytest.mark.parametrize("model_class, parameters", MODELS_AND_DATASETS)
+# Neural network hyper-parameters are not tested
 @pytest.mark.parametrize(
-    "n_bits",
-    N_BITS_WEEKLY_ONLY_BUILDS + N_BITS_REGULAR_BUILDS,
```

Comment:
not exactly sure why but `get_hyper_param_combinations` does not provide anything for QNNs, meaning they were not tested here at all